Source/Core/src/ca/uqac/lif/labpal/server/ExplainImageCallback.java

```diff
@@ -22,106 +22,97 @@
-	{
+public class ExplainImageCallback extends WebCallback {
+	public ExplainImageCallback(Laboratory lab, LabAssistant assistant) {
 		super("/provenance-graph", lab, assistant);
 	}
 
 	@Override
-	public CallbackResponse process(HttpExchange t)
-	{
+	public CallbackResponse process(HttpExchange t) {
 		CallbackResponse response = new CallbackResponse(t);
-		Map<String,String> params = getParameters(t);
+		Map<String, String> params = getParameters(t);
 		String datapoint_id = params.get("id");
 		ProvenanceNode node = m_lab.getDataTracker().explain(datapoint_id);
-		if (node == null)
-		{
-			response.setContents("<html><body><h1>Not Found</h1><p>This data point does not seem to exist.</p></body></html>");
+		if (node == null) {
+			response.setContents(
+					"<html><body><h1>Not Found</h1><p>This data point does not seem to exist.</p></body></html>");
 			response.setCode(CallbackResponse.HTTP_NOT_FOUND);
 			return response;
 		}
 		DotProvenanceTreeRenderer renderer = new DotProvenanceTreeRenderer();
-		if (!GraphvizRenderer.s_dotPresent)
-		{
+		if (!GraphvizRenderer.s_dotPresent) {
 			// Asking for an image, but DOT not available: stop right here
-			response.setContents("<html><body><h1>Not Found</h1><p>DOT is not present on this system, so the picture cannot be shown.</p></body></html>");
+			response.setContents(
+					"<html><body><h1>Not Found</h1><p>DOT is not present on this system, so the picture cannot be shown.</p></body></html>");
 			response.setCode(CallbackResponse.HTTP_NOT_FOUND);
 			return response;
 		}
 		String extension = "svg";
 		response.setContentType("image/svg+xml");
-		if (params.containsKey("format") && params.get("format").compareToIgnoreCase("pdf") == 0)
-		{
+		if (params.containsKey("format") && params.get("format").compareToIgnoreCase("pdf") == 0) {
 			response.setContentType(ContentType.PDF);
 			extension = "pdf";
 		}
 		byte[] image = renderer.toImage(node, extension);
-		if (image == null)
-		{
-			response.setContents("<html><body><h1>Internal Server Error</h1><p>The image cannot be displayed.</p></body></html>");
+		if (image == null) {
+			response.setContents(
+					"<html><body><h1>Internal Server Error</h1><p>The image cannot be displayed.</p></body></html>");
 			response.setCode(CallbackResponse.HTTP_INTERNAL_SERVER_ERROR);
-			return response;			
+			return response;
 		}
 		response.setContents(image);
 		response.setCode(CallbackResponse.HTTP_OK);
-		if (params.containsKey("dl"))
-		{
+		if (params.containsKey("dl")) {
 			response.setAttachment(Server.urlEncode(node.getNodeFunction().getDataPointId() + "." + extension));
 		}
 		return response;
 	}
 
 	@Override
 	public void addToZipBundle(ZipOutputStream zos) throws IOException {
-	
-	/*	Set<Integer> ids = m_lab.getTableIds();
-		for (int id : ids) {
-			Table tab = m_lab.getTable(id);
-			HardTable tbl = tab.getDataTable();
-			render1(zos,tab, tbl.getTree(), tbl.getColumnNames());
-			zos.closeEntry();
-		}*/
-		
+
```

Comment:
The indentation for this commented bit of code has been completely messed up. Please restore it to what it was.